doc/INSTALL.md

```diff
@@ -230,27 +230,30 @@ Checkout and Build PIConGPU
 
 This is an example how to use the modular building environment of PIConGPU.
 
-1. **Setup directories:** `mkdir -p ~/src ~/build ~/paramSets ~/runs`
-   - `~/runs` is the directory for PIConGPU simulation output
+1. **Setup PIConGPU home** `export PICHOME=~/`
+   - Note that the PICHOME path can be set to any other location.
+
+2. **Setup directories:** `mkdir -p $PICHOME/src $PICHOME/build $PICHOME/paramSets $PICHOME/runs`
+   - `$PICHOME/runs` is the directory for PIConGPU simulation output
    - NOTE for HPC-Systems: Never write your simulation output to your home
      (=`~/`) directory
```

Comment:
(`~/`, $HOME) directory but ask your admin about the mount point of your *dedicated high-bandwidth file-system*